monitor/alerts.py

```diff
@@ -30,8 +30,15 @@ def send(webhook: str, data: dict, content_types: str = "application/json"):
     def to_slack_message(self) -> dict:
         pass
 
-    def send_to_slack(self, webhook: str):
-        data = self.to_slack_message()
+    def to_slack_workflows_message(self) -> dict:
+        pass
+
+    def send_to_slack(self, webhook: str, is_slack_workflow = False):
```

Comment:
small cosmetics - add bool type to is_slack_workflow

Reply:
[adding bool typing to signature](https://github.com/elementary-data/elementary/pull/54/commits/71e8c8db1e59c4330922f975e840497fd311d184)